hr_attendance_rest_time_included/models/hr_attendance_rest_time.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Tecnativa - Eduardo Ezerouali
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import fields, models
+
+
+class HrAttendanceRestTime(models.Model):
+    _inherit = "hr.attendance"
+    _name = "hr.attendance.rest_time"
+    _description = "Hr attendance break"
+
+    hr_attendance_id = fields.Many2one(
```

Comment:
This is usually put this way, as it doesn't add too much value to put `hr_` as prefix, but this is not blocking:

```suggestion
    attendance_id = fields.Many2one(
```